client/src/components/organisms/DonorImpactSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FunctionComponent, useState } from "react";
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import MapWithMarkers, { MapWithMarkersPoint } from '../atoms/MapWithMarkers'
+import { Statistic, Testimonial } from '../../data/types/donorHomepageConfig'
+import BaseMap from '../../assets/kw-region-map.png'
+import Card from '../atoms/CardWithShadow'
+import Cursor from '../../assets/cursor.png'
+import DonorHomepageConfig from '../../config/donorHompageConfig.json'
+
+const DonorImpactSection: FunctionComponent = () => {
+  const [selectedTestimonial, _setSelectedTestimonial] = useState<Testimonial | null>(null);
+  const testimonials = DonorHomepageConfig.map.testimonials;
+  const setSelectedTestimonial = (id: number | null) => {
+    if (!id) {
+      _setSelectedTestimonial(null);
+      return;
+    }
+    const testimonial = testimonials.find((testimonial: Testimonial) => testimonial.id === id);
+    _setSelectedTestimonial(testimonial ?? null);
+  }
+  const points = DonorHomepageConfig.map.points;
+  const markerSize = testimonials.length > 0 && testimonials.length <= DonorHomepageConfig.map.markerSizes.length
+    ? DonorHomepageConfig.map.markerSizes[testimonials.length - 1]
+    : DonorHomepageConfig.map.defaultMarkerSize;
+  const maxPoints = points.length;
```

Comment:
Nit, don't need a variable for a simply computed property that will need to update on each state change anyways, just use points.length where maxPoints is used for clarity

Reply:
points.length is labelled as maxPoints so that readers know that points.length caps the number of testimonials that can be displayed

this is better than including a comment where points.length is used because the code itself is self-documenting and no one will have to maintain the comment